### ISSUE TITLE
Is there something wrong with the config in resnet of mpii?

### ISSUE THREAD

**Issue by ElegantLin** [AIprogrammer/AdvMix#3]
Here is the test part of `res50_256x256_d256x3_adam_lr1e-3_advmix.yaml`. Why is there a key called `COCO_BBOX_FILE`? Is there something wrong with this part?

Thanks!

```
TEST:
  BATCH_SIZE_PER_GPU: 32
  COCO_BBOX_FILE: 'data/coco/person_detection_results/COCO_val2017_detections_AP_H_56_person.json'
  BBOX_THRE: 1.0
  IMAGE_THRE: 0.0
  IN_VIS_THRE: 0.2
  MODEL_FILE: ''
  NMS_THRE: 1.0
  OKS_THRE: 0.9
  FLIP_TEST: true
  POST_PROCESS: true
  SHIFT_HEATMAP: true
  USE_GT_BBOX: true
```

**Comment by jin-s13** [comment]
Some keys are not used.

**Comment by ElegantLin** [comment]
Thanks. Let me try again. If there is no issue, I would close this.
